### ISSUE TITLE
Minimap on blank

### ISSUE THREAD

**Issue by BigSlammu** [ValveSoftware/Dota2-Gameplay#7694]
### Cosmetic Item Name

Minimap

### Description

i log my dota and when i start an match the minimap are on blank i close and reopen my game and still same problem with bots

### Example Match ID

no idea

**Comment by shaileshgarg** [comment]
"Hide Minimap Background" in Options, it is on?

**Comment by BigSlammu** [comment]
no, the crazy is when u choice an hero u had the minimap but after all heros in to the match literally get blank on solid grey

**Comment by shaileshgarg** [comment]
You should add a Video showing the issue, also what OS are you using?
